clients/ui/frontend/src/app/pages/modelCatalog/screens/ModelCatalogSourceLabelSelector.tsx

```diff
@@ -72,9 +86,14 @@ const ModelCatalogSourceLabelSelector: React.FC<ModelCatalogSourceLabelSelectorP
     <Stack>
       <StackItem>
         <Toolbar
+          key={`toolbar-${hasActiveFilters}`}
           className="pf-v6-u-pb-0"
-          clearAllFilters={onResetAllFilters}
-          clearFiltersButtonText="Reset all filters"
+          {...(onResetAllFilters
```

Comment:
These conditionals for `onResetAllFilters` aren't doing anything, that prop doesn't need to be optional. It's always passed.